woocommerce/woocommerce
# Add e2e tests to verify block templates can be customized if the theme has its own custom templates

### Changes proposed in this Pull Request:

This PR:

1. Creates a new Playwright test environment so we run tests in a block theme with custom templates.
2. In that environment, it adds a test for each template to verify they can be customized.
3. It updates some `package.json` scripts and some docs that were not up to date.
4. Removes `plugins/woocommerce-blocks/.github/workflows/playwright.yml`, as I understand, that file has been superseded by `.github/workflows/blocks-playwright.yml`.

Part of #43416.

### How to test the changes in this Pull Request:

Note: no need to test this for the release.

1. Verify e2e tests pass and there is a job specific for tests run in a block theme with templates.
2. Locally run `npm run test:e2e:side-effects`, `npm run test:e2e:side-effects:report`, `npm run test:e2e:classic-theme`, `npm run test:e2e:classic-theme:report`, `npm run test:e2e:block-theme-with-templates` and `npm run test:e2e:block-theme-with-templates:report` and verify all of them work (the report ones should open the report in your browser once finished).
3. Read the updated docs and verify they make sense.

### Changelog entry


<details>

#### Significance


#### Type


#### Message 

#### Comment

Add e2e tests to verify block templates can be customized if the theme has its own custom templates.

</details>
